metricbeat/module/windows/perfmon/_meta/data.json

```diff
@@ -1,19 +1,24 @@
 {
-    "@timestamp":"2016-05-23T08:05:34.853Z",
-    "beat":{
-        "hostname":"beathost",
-        "name":"beathost"
+    "@timestamp": "2016-05-23T08:05:34.853Z",
+    "beat": {
+        "hostname": "host.example.com",
+        "name": "host.example.com"
     },
-    "metricset":{
-        "host":"localhost",
-        "module":"mysql",
-        "name":"status",
-        "rtt":44269
+    "metricset": {
+        "module": "windows",
+        "name": "perfmon",
+        "rtt": 115
     },
-    "windows":{
-        "perfmon":{
-            "example": "perfmon"
+    "type": "metricsets",
+    "windows": {
+        "perfmon": {
+            "processor": {
+                "time": {
+                    "total": {
+                        "pct": null
```

Comment:
BTW: Could you perhaps add at least 2-3 values in your config to have here multiple values?